insights/parsers/postfix_conf.py

```diff
@@ -0,0 +1,81 @@
+"""
+PostfixMaster - file ``/etc/postfix/master.cf``
```

Comment:
Please double confirm the title and the name of the python file. Did you use different names on purpose?